emission/core/wrapper/compositetrip.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import emission.core.wrapper.confirmedtrip as ecwc
+import emission.core.wrapper.wrapperbase as ecwb
+import emission.core.wrapper.modeprediction as ecwm
+
+class Compositetrip(ecwc.Confirmedtrip):
+    props = ecwc.Confirmedtrip.props
+    props.update({#      # confirmedplace stuff
+                  "confirmed_place": ecwb.WrapperBase.Access.WORM, # object contains all properties for the destination confirmed_place object
+                  #      # sections stuff
+                  "cleaned_section": ecwb.WrapperBase.Access.WORM,
+                  "inferred_section": ecwb.WrapperBase.WORM,
+                  "inferred_mode": ecwb.WrapperBase.WORM, # inferred by mode inference algo
+                  "confirmed_mode": ecwb.WrapperBase.WORM, # confirmed by user
```

Comment:
we do want to have the trajectory here as well in the future

Reply:
Let's make this a to-do item